pull_request_template.md

```diff
@@ -1,9 +1,10 @@
 <!-- First of all, THANK YOU for your contribution. -->
 
 ## Changes Proposed:
--  [ ] This pull request promotes changes to the core (units, players, creatures, game systems).
--  [ ] This pull request promotes changes to scripts (bosses, spell scripts, creature scripts).
--  [ ] This pull request promotes database changes.
+This PR proposes changes to:
+-  [ ] The core (units, players, creatures, game systems).
+-  [ ] To scripts (bosses, spell scripts, creature scripts).
+-  [ ] To the database (SAI, creatures, etc).
```

Comment:
Would make more grammatical sense as the following for example:
```
The core
Scripts
The database
```
Otherwise a full sentence would be "this PR proposes changes to to scripts".